tests/Integration/Ingester/Ingester/UserIngesterTest.php

```diff
@@ -125,6 +132,29 @@ public function testIngestWithValidSource(): void
         $this->assertEquals('user_12', $user12->getUsername());
     }
 
+    public function testItCanIngestUsersWithGroupId(): void
+    {
+        $this->prepareIngestionContext();
+
+        $source = $this->createIngesterSource(__DIR__ . '/../../../Resources/Ingester/Valid/users-with-groupId.csv');
+
+        $output = $this->subject->ingest($source, false);
+
+        $this->assertEquals(12, $output->getSuccessCount());
+        $this->assertFalse($output->hasFailures());
+
+        $this->assertCount(12, $this->getRepository(User::class)->findAll());
+
+        $usersInGroup1 = $this->getRepository(User::class)->findBy(['groupId' => 'group_1']);
+        $this->assertCount(3, $usersInGroup1);
+
+        $usersInGroup2 = $this->getRepository(User::class)->findBy(['groupId' => 'group_2']);
+        $this->assertCount(4, $usersInGroup2);
+
+        $usersInGroup3 = $this->getRepository(User::class)->findBy(['groupId' => 'group_3']);
+        $this->assertCount(5, $usersInGroup3);
+    }
```

Comment:
$usersInGroup4 = $this->getRepository(User::class)->findBy(['groupId' => 'group_4']);

$usersInGroup4 === ?

Reply:
It's ok as we discussed